src/hooks/dialogmote/useForhandsvisInnkalling.ts

```diff
@@ -46,10 +49,11 @@ export const useForhandsvisInnkalling = (): ForhandsvisInnkallingGenerator => {
 
   const generateArbeidstakerInnkallingDocument = (
     values: Partial<DialogmoteInnkallingSkjemaValues>,
+    valgtNarmesteLeder: string,
     valgtBehandler?: BehandlerDialogmeldingDTO
   ) => {
     const documentComponents = [
```

Comment:
Siden vi allerede har valgt virksomhetsnummer i `values` her så kan man bruke `getCurrentNarmesteLeder` fra `useLedere`  for å hente valgt virksomhetsnavn her og sende det til fellesinfo. 
Så slipper man egen state for det i komponenten og ekstra parameter i generate-funksjonene.

Reply:
men jeg kan vel ikke slenge inn en hook inni den funksjonen?

Reply:
Tenkte å noe sånt i `useForhandsvisInnkalling` (før definisjonen av generate-funksjonene):
```
const { getCurrentNarmesteLeder } = useLedere();
const getValgtArbeidsgiver = (values: Partial<DialogmoteInnkallingSkjemaValues>) => values.arbeidsgiver && getCurrentNarmesteLeder(values.arbeidsgiver)?.virksomhetsnavn; // evt tom string hvis undefined
```
Så kan `getValgtArbeidsgiver` brukes inni generate* f.eks.